cypress/page/address-step.page.ts

```diff
@@ -2,7 +2,9 @@ class AdressStepPage {
     private proceedToCheckoutAdress: string;         
 
     constructor() {        
-        this.proceedToCheckoutAdress = ".cart_navigation span";
+        this.proceedToCheckoutAdress = "[type='submit'][name='processAddress']";
+	    // Here, this selector can be improved. Using of classes is avoided,
+        // attributes are good selectors.
```

Comment:
justification comments are not needed here, we we talk about justify your selection are only for the cases we comment in the PR to change are you are not agree 😉